packages/canvas-graphics/src/CanvasGraphicsRenderer.ts

```diff
@@ -96,9 +96,7 @@ export class CanvasGraphicsRenderer
         let contextFillStyle;
         let contextStrokeStyle;
 
-        const tintR = ((graphics.tint >> 16) & 0xFF) / 255;
-        const tintG = ((graphics.tint >> 8) & 0xFF) / 255;
-        const tintB = (graphics.tint & 0xFF) / 255;
+        const tint = Color.shared.setValue(graphics.tint).toArray();
```

Comment:
Anything that touches rendering is hot code. In this case, this is to avoid extra garbage, as using `shared` is the same thing as using one of those static color util functions. We don't need to be instantiating `new Color` 60fps.